packages/v2/gui/src/sak/meny/marker-behandling/MarkerBehandlingBackendClient.ts

```diff
@@ -0,0 +1,49 @@
+import type {
+  DeleteMerknadResponse,
+  EndreMerknadRequestMerknadKode,
+  GetMerknadResponse,
+  K9SakClient,
+  PostMerknadResponse,
+  SlettMerknadRequestMerknadKode,
+} from '@k9-sak-web/backend/k9sak/generated';
+
+export default class MarkerBehandlingBackendClient {
+  #backendClient: K9SakClient;
+
+  constructor(backendClient: K9SakClient) {
+    this.#backendClient = backendClient;
+  }
+
+  async getMerknader(behandlingUuid: string): Promise<GetMerknadResponse> {
+    return this.#backendClient.los.getMerknad(behandlingUuid);
```

Comment:
Kan det by på problemer at metode signaturen indikerer at den returnerer en `Promise<GetMerknadResponse>`, men `LosService.getMerknad()` returnerer en `CancelablePromise<GetMerknadResponse>`? Burde enten bruke samme signatur eller kanskje resolve Promise?

Reply:
samme for `postMerknad()` og `deleteMerknad()`